.gitignore

```diff
@@ -30,9 +30,10 @@ tests/test*/*
 playground/*
 tutorial/*
 
-snakemake/assets/data
+src/snakemake/assets/data
+src/snakemake/_version.py
```

Comment:
`seuptools-scm`'s `_version.py` file is dynamically generated, and so should never get added to version control.